keepassxc-browser/content/passkeys.js

```diff
@@ -85,6 +85,38 @@ const handleError = function(errorCode, errorMessage) {
     throw new DOMException(errorMessage, 'UnknownError');
 };
 
+// Create a real PublicKeyCredential object
+const createPublicKeyCredential = function(publicKeyOptions) {
+    const publicKey = Object.create(PublicKeyCredential.prototype, {
+        authenticatorAttachment: {
+            value: publicKeyOptions.authenticatorAttachment
+        },
+        clientExtensionResults: {
+            value: () => {}
+        },
+        getClientExtensionResults: {
+            value: () => {}
+        },
+        id: {
+            value: publicKeyOptions.id
+        },
+        isConditionalMediationAvailable: {
```

Comment:
I don't think this is right as it's supposed to be a static property, i.e. it should be on the prototype rather than the object instance, as it's used like `PublicKeyCredential.isConditionalMediationAvailable()`, not `publicKey.isConditionalMediationAvailable()` (where `publicKey` is an instance of `PublicKeyCredential`). I think the `Object.defineProperty()` you've already got for this property further down is enough.

Reply:
Ah, got it. I was about to remove the `Object.defineProperty()` one though, but maybe it's better to keep this there.